wis2node/util.py

```diff
@@ -86,6 +86,24 @@ def json_serial(obj: object) -> Union[bytes, str, float]:
     raise TypeError(msg)
 
 
+def walk_path(path: str, regex: str) -> list:
```

Comment:
Update function annotation given this function (you will need to `from typing import Iterator`) like:

`def walk_path(path: str, regex: str) -> Iterator[str]:`